examples/recipes/robot_navigate.py

```diff
@@ -0,0 +1,18 @@
+from pitop import DriveController, NavigationController
+
+drive_controller = DriveController(left_motor_port="M3", right_motor_port="M0")
+
+navigation_controller = NavigationController(drive_controller=drive_controller)
```

Comment:
Why not have an internal drive controller? Can't see any use in declaring it outside...
```suggestion
from pitop import NavigationController

navigation_controller = NavigationController(left_motor_port="M3", right_motor_port="M0")
```

Reply:
we're using the drive controller inside the navigation controller - I'd rather not duplicate that code. 

Also, there are many situations where a user would want to add both a drive controller and a navigation controller to their robot. E.g. use `robot.drive.target_lock_drive_angle()` to go and get a coloured ball, then they would switch to navigation controller to go and drop it off at a certain location. In this case we want to pass in DriveController directly so that they don't have to duplicate port assignment